modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -150,49 +87,6 @@ def get_benefits_documents_auth_token
         @auth_token ||= ClaimsApi::V2::BenefitsDocuments::Service.new.get_auth_token
       end
 
-      def build_target_veteran(veteran_id:, loa:) # rubocop:disable Metrics/MethodLength
-        target_veteran ||= ClaimsApi::Veteran.new(
-          mhv_icn: veteran_id,
-          loa:
-        )
-        # populate missing veteran attributes with their mpi record
-        found_record = target_veteran.mpi_record?(user_key: veteran_id)
-
-        unless found_record
-          claims_v2_logging('veteran_not_found', icn: veteran_id,
-                                                 message: 'Claims v2 Veteran record not found')
```

Comment:
do we want the logging lines added back in to the `target_veteran.rb`? I feel like their may be usage for it? maybe not though..